DVSAdmin.BusinessLogic/Services/RegManagement/RegManagementService.cs

```diff
@@ -128,17 +128,22 @@ public async Task<GenericResponse> UpdateServiceStatus(List<int> serviceIds, int
         /// <returns></returns>
         public async Task<GenericResponse> UpdateRemovalStatus(EventTypeEnum eventType,  int providerProfileId, List<int> serviceIds, string loggedInUserEmail, List<string> dsitUserEmails, RemovalReasonsEnum? reason, ServiceRemovalReasonEnum? serviceRemovalReason)
         {
-            TeamEnum team = eventType == EventTypeEnum.RemovedByCronJob ? TeamEnum.CronJob : TeamEnum.DSIT;
-            GenericResponse genericResponse = await regManagementRepository.UpdateRemovalStatus(eventType, team, providerProfileId, serviceIds, loggedInUserEmail, reason, serviceRemovalReason);
-
-            if (genericResponse.Success)
-            {
+            GenericResponse genericResponse = await regManagementRepository.UpdateRemovalStatus(eventType, TeamEnum.DSIT, providerProfileId, serviceIds, loggedInUserEmail, reason, serviceRemovalReason);
 
+                TeamEnum requstedBy = TeamEnum.DSIT;
+                if (reason == RemovalReasonsEnum.ProviderRequestedRemoval || (serviceRemovalReason!=null && serviceRemovalReason == ServiceRemovalReasonEnum.ProviderRequestedRemoval))
+                {
+                    requstedBy = TeamEnum.Provider;
```

Comment:
Can "requstedBy" be refactored to "requestedBy"?
```suggestion
                    requestedBy = TeamEnum.Provider;
```